app/views/pages/changelog.cs-CZ.html.haml

```diff
@@ -0,0 +1,265 @@
+#changelog
+  %h1 Co je nového?
+  %p
+    Toto je přehled nedávných změn na tété stránce.
+    Kompletní detailní historii vývoje můžete sledovat na
+    = t(".github_history_link_html")
+  %article
```

Comment:
I think we should just split body of `changelog.html.haml` so https://github.com/cyclestreets/cyclescape/blob/master/app/views/pages/changelog.html.haml#L7 and here would become:
```ruby
  = render "changes"
```
and then move all everything below this line into `app/views/pages/_changes.html.haml`.  Does that make sense?